common/src/main/java/com/microsoft/identity/common/internal/cache/ActiveBrokerCacheUpdater.kt

```diff
@@ -125,6 +131,7 @@ class ActiveBrokerCacheUpdater(
         val signingCertThumbprint = bundle.getString(ACTIVE_BROKER_SIGNING_CERTIFICATE_THUMBPRINT_KEY)
 
         if (packageName.isNullOrEmpty() || signingCertThumbprint.isNullOrEmpty()) {
+            Logger.info(methodTag, "A response was received without active broker information.")
```

Comment:
Should we also log if it was requested to begin with?

Reply:
done